### PR TITLE
ci(gh-action): use new action notify-slack-action

### PR DESCRIPTION
**Pull request by akantcheff**

### CODE REVIEW

File: .github/workflows/propagate-doc-upwards.yml [hunk 1]
```diff
@@ -56,13 +56,13 @@ jobs:
 
       - name: Send message to Slack channel
         if: failure() && (github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' && github.event.inputs.slack-notifications == 'true') )
-        uses: bonitasoft/actions/packages/notify-slack@v2
+        uses: bonitasoft/notify-slack-action@v1
         with:
-          CHANNEL_ID: ${{ vars.SLACK_CHANNEL_ID }}
-          MESSAGE: |
+          keeper-secret-config: ${{ secrets.KSM_CONFIG }}
+          channel-id: ${{ vars.DEVSUITE_CORE_SLACK_CHANNEL_ID }}
```
Comment (akantcheff) [inline .github/workflows/propagate-doc-upwards.yml:62]: Oops

File: .github/workflows/propagate-doc-upwards.yml [hunk 2]
```diff
@@ -56,13 +56,13 @@ jobs:
 
       - name: Send message to Slack channel
         if: failure() && (github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' && github.event.inputs.slack-notifications == 'true') )
-        uses: bonitasoft/actions/packages/notify-slack@v2
+        uses: bonitasoft/notify-slack-action@v1
         with:
-          CHANNEL_ID: ${{ vars.SLACK_CHANNEL_ID }}
```
Comment (akantcheff) [inline .github/workflows/propagate-doc-upwards.yml:61]: Delete this var after merging

**Review by benjaminParisel — APPROVED** [review]
:ok_hand:

**Comment by benjaminParisel** [comment]
Variable `SLACK_CHANNEL_ID` removed 
image

**Pull request merged**